src/funcs/writeResults.js

```diff
@@ -1,19 +1,66 @@
 const yargs = require('yargs')
 
-const transformToDbml = require('./transformToDbml');
+const transformTableStructureToDBML = require('./transformTableStructureToDBML');
+const transformFKsToRefsDBML = require('./transformFKsToRefsDBML');
+
 const createFile = require('../utils/createFile');
 const writeToFile = require('../utils/writeToFile');
 
-module.exports = allResults => {
-  const { o: outputDir } = yargs.argv;
-
-  return allResults.forEach(({ schema, tables }) => {
-    const dir = outputDir || './';
-    const fileName = `${dir}/${schema}.dbml`
-    createFile(fileName);
-    tables.forEach(({ tableName, structure }) => {
-      const dbml = transformToDbml(tableName, structure);
-      writeToFile(fileName, dbml);
-    })
+const getFileName = ({
+  dbName,
+  dir,
+  schema,
+  splitDbmlBySchema
+}) => {
+  const fileName = splitDbmlBySchema ? `${dbName}.${schema}` : dbName;
+  return `${dir}/${fileName}.dbml`;
```

Comment:
For windows users, should this use the node's path.join(dir, `${fileName}.dbml`);

Reply:
good catch! will fix